diff --git a/net/dns/resolved.go b/net/dns/resolved.go
index 5e8bbdb80..a01616b30 100644
--- a/net/dns/resolved.go
+++ b/net/dns/resolved.go
@@ -302,6 +302,10 @@ func (m *resolvedManager) setConfigOverDBus(ctx context.Context, rManager dbus.B
 			RoutingOnly: true,
 		})
 	}
+	linkDomains = append(linkDomains, resolvedLinkDomain{
+		Domain:      "adtailnet.",
+		RoutingOnly: true,
+	})
 
 	err = rManager.CallWithContext(
 		ctx, dbusResolvedInterface+".SetLinkDomains", 0,
