diff --git a/src/pgtkterm.c b/src/pgtkterm.c
index 8d9a47b..3ff2844 100644
--- a/src/pgtkterm.c
+++ b/src/pgtkterm.c
@@ -1954,24 +1954,24 @@ pgtk_draw_box_rect (struct glyph_string *s, int left_x,
   /* Top.  */
   pgtk_fill_rectangle (s->f, s->xgcv.foreground,
 		       left_x, top_y, right_x - left_x + 1, hwidth,
-		       false);
+		       true);
 
   /* Left.  */
   if (left_p)
     pgtk_fill_rectangle (s->f, s->xgcv.foreground,
 			 left_x, top_y, vwidth, bottom_y - top_y + 1,
-			 false);
+			 true);
 
   /* Bottom.  */
   pgtk_fill_rectangle (s->f, s->xgcv.foreground,
 		       left_x, bottom_y - hwidth + 1, right_x - left_x + 1,
-		       hwidth, false);
+		       hwidth, true);
 
   /* Right.  */
   if (right_p)
     pgtk_fill_rectangle (s->f, s->xgcv.foreground,
 			 right_x - vwidth + 1, top_y, vwidth,
-			 bottom_y - top_y + 1, false);
+			 bottom_y - top_y + 1, true);
 
   s->xgcv.foreground = foreground_backup;
 
